apps/yaft/keyboard.cpp

```diff
@@ -288,6 +300,23 @@ getKeyCodeStr(int scancode, bool shift, bool alt, bool ctrl, bool appCursor) {
     buf[4] = 0;
   };
 
+  constexpr auto write_f_code = [](char code) {
+    buf[0] = esc_char;
+    buf[1] = '[';
+    buf[2] = '[';
+    buf[3] = code;
+    buf[4] = 0;
+  };
+
+  constexpr auto write_hi_f_code = [](char a, char b) {
+    buf[0] = esc_char;
+    buf[1] = '[';
+    buf[2] = a;
+    buf[3] = b;
+    buf[4] = '~';
+    buf[5] = 0;
+  };
+
```

Comment:
From https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-PC-Style-Function-Keys
```suggestion
  constexpr auto write_ss3 = [](char code) {
    buf[0] = esc_char;
    buf[1] = 'O';
    buf[3] = code;
    buf[4] = 0;
  };

  constexpr auto write_hi_vt_code = [](char a, char b) {
    buf[0] = esc_char;
    buf[1] = '[';
    buf[2] = a;
    buf[3] = b;
    buf[4] = '~';
    buf[5] = 0;
  };

```